testing/tests/optional/markdown.rs

```diff
@@ -0,0 +1,73 @@
+use askama::Template;
```

Comment:
We can just have `#![cfg()]` in here instead of the additional module?

Reply:
Then everything needs to be annotated or everything needs to go into a ↓ block.

```rs
#[cfg(…)]
const _: () = {
    …
};
```

This would work for me, too, but I like the "optional" module better, actually. I would rather remove the `#[cfg()]`s from the other places in the testing module, and move the tests into "optional".

What do you think? Is this "over engineered"?

Reply:
I think you missed the `!` -- it's an inner attribute, that applies to the module as a whole.

Reply:
Ahhhhh, yes, I did, because I simply assumed that disabling a whole file cannot be possible. Will change.